apis/swagger.yml

```diff
@@ -551,6 +551,26 @@ paths:
           $ref: "#/responses/500ErrorResponse"
       tags: ["Container"]
 
+  /containers/{id}/kill:
+    post:
+      summary: "Kill a container"
+      operationId: "ContainerKill"
+      parameters:
+        - $ref: "#/parameters/id"
+        - name: "signal"
+          in: "query"
+          description: "signal to send to the container"
+          type: "string"
+          default: "SIGKILL"
+      responses:
+        204:
+          description: "no error"
```

Comment:
You add status code 400 in code, then here we need to add 400 as well in swagger. @xiechengsheng